src/coreclr/src/gc/gcsvr.cpp

```diff
@@ -21,6 +21,10 @@
 
 #define SERVER_GC 1
 
+#if defined(TARGET_AMD64)
```

Comment:
I suppose you might as well also change this to check for `TARGET_WINDOWS` to be consistent.

Reply:
Right, that slipped throught the cracks - thanks!